doc/rfcs/003-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -0,0 +1,147 @@
+    - Teletype would need additional permissions (i.e., [OAuth scopes](https://developer.github.com/apps/building-oauth-apps/scopes-for-oauth-apps/#available-scopes)) to fetch the list of users you're associated with. (Today, Teletype  uses a scopeless token and has no access to your private information.)
+    - Some users belong to organizations with thousands of members. Just because you're a member of the same organization as someone else doesn't mean that you're comfortable seeing portal invitations from them.
+- Teletype could add support for audio before adding support for inviting past collaborators to your portal. When collaborating with people in different physical locations, you'll likely need audio support in order to collaborate. Since Teletype doesn't currently provide audio support, these collaborators will still need a third-party solution for audio. If you're already relying on a third-party solution for audio, you can often use that same solution to send your collaborators your portal ID, which reduces the need for Teletype providing a list of past collaborators. However, there are still many [scenarios where it's helpful to quickly invite a past collaborator even if Teletype doesn't yet support audio](https://github.com/atom/teletype/pull/344#discussion_r175319913).
+
+##### What is the impact of not doing this?
+
+People will collaborate less often. Given the additional steps needed to start collaborating, there will be more instances where people decide that it's not worth the effort (i.e., people will enjoy rich collaboration less often).
+
+## Unresolved questions
+
+##### What unresolved questions do you expect to resolve through the RFC process before this gets merged?
+
+- If I invite a past collaborator to join my portal, and that person has multiple Atom windows open, can we show the invitation only in the frontmost window?
+- If I invite a past collaborator to join my portal, and that person has Atom windows open on multiple computers, are we OK with showing the invitation on each computer?
+- Where will the list of past collaborators appear in the UI? How do I show/hide this list?
+
+##### What unresolved questions do you expect to resolve through the implementation of this feature before it is released in a new version of the package?
+
+- What architecture/services/libraries will we use to implement presence?
+- In order to provide the functionality described above, does teletype-server need to persist your list of past collaborators, or can we meet these needs while only storing this data locally?
+- If the host invites a past collaborator to their portal and then closes the portal before the invitee joins the portal, what should an invitee see/experience to inform them that the portal no longer exists?
```

Comment:
I think modeling this invitation interaction on the server could once again solve this. Upon disconnection, the server could query all the invitations from the disconnecting user and mark their invitations as expired.